MHacks/RoundedView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RoundedView.swift
+//  MHacks
+//
+//  Created by Connor Krupp on 9/11/16.
+//  Copyright © 2016 MHacks. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class RoundedView: UIView {
```

Comment:
@russellladd That's what I did first but I tried it on simulator and on device and I can't just can't see any rounding unless I put 2 :/